rust/openvasd-tests/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# to be built with openvas-scanner top level dir as base
+
+FROM registry.community.greenbone.net/community/gvm-libs:edge
+
+RUN apt-get update && apt-get install --no-install-recommends --no-install-suggests -y \
```

Comment:
is there a reason for not using install-openvas-dependencies?

https://github.com/greenbone/openvas-scanner/blob/main/.devcontainer/Dockerfile#L40